e2e-test/src/test/features/GroupByMacros.feature

```diff
@@ -0,0 +1,59 @@
+@GroupBy
+Feature:GroupBy - Verification of GroupBy pipeline with GCS as source and File as target using macros
+
+  @GCS_CSV_GROUPBY_TEST @FILE_SINK_TEST
+  Scenario: To verify data is getting transferred from GCS to File successfully with GroupBy plugin properties as macro arguments
```

Comment:
Can we use File Source and sink in all scenarios ? so that there wont be rework incase.

Reply:
Used only File as source and File as sink in scenarios.